backend/src/db/migrations/20250418003930_ssh-nullable-ca-defaults.ts

```diff
@@ -0,0 +1,55 @@
+import { Knex } from "knex";
+
+import { ProjectType, TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  const hasDefaultUserCaCol = await knex.schema.hasColumn(TableName.ProjectSshConfig, "defaultUserSshCaId");
+  const hasDefaultHostCaCol = await knex.schema.hasColumn(TableName.ProjectSshConfig, "defaultHostSshCaId");
+
+  if (hasDefaultUserCaCol && hasDefaultHostCaCol) {
+    await knex.transaction(async (trx) => {
+      await trx.schema.alterTable(TableName.ProjectSshConfig, (t) => {
```

Comment:
Migration is already tx, any reason for another transaction?

Reply:
Removed.